common/src/warp_runner/manager/commands/constellation_commands.rs

```diff
@@ -263,6 +270,10 @@ async fn rename_item(
     get_items_from_current_directory(warp_storage)
 }
 
+fn get_storage_size(warp_storage: &mut warp_storage) -> Result<(usize, usize), Error> {
```

Comment:
No need for mutable borrow if the calls are immutable
```suggestion
fn get_storage_size(warp_storage: &warp_storage) -> Result<(usize, usize), Error> {
```